Documentation/operations/troubleshooting.rst

```diff
@@ -713,6 +713,40 @@ When running in :ref:`arch_direct_routing` mode:
 
 4. Verify that the firewall on each node permits to route the endpoint IPs.
 
+Traffic to Pods of a Service is unbalanced
+------------------------------------------
+
+Symptom
+~~~~~~~
+
+Endpoint to endpoint communication between nodes is successful, but connections
+and / or load appears unbalanced to one or more Pods that make up a Service.
+One or more Pods receives more traffic than the others.
+
+Troubleshooting steps:
+~~~~~~~~~~~~~~~~~~~~~~
+
+Confirm the load balancing mode enabled for Services in the cluster, set by the
+``hostServices.enabled`` Helm flag or the ``enable-host-reachable-services``
+flag passed to the Cilium agent. More information found on Host-Reachable
+services in :ref:`host-services`.
+
+If host-reachable services are disabled, (i.e.
+``hostServices.enabled=false``), the following parameters can provide more
+fine-grained control over service load balancing in the cluster:
+
+  * ``bpf-ct-timeout-service-*`` - The amount of time endpoints are tracked via
+    conntrack becoming eligible for garbage collection.
+  * ``conntrack-gc-interval`` - The garbage collection period for conntrack.
+
+Note: tuning the above can have an adverse impact on CPU usage. More frequent
+GC can result in a more even distribution of traffic, but comes at the cost of
+CPU usage.
+
+Another option is to consider whether running without ``kube-proxy`` is an
+option for your cluster. This feature makes use of the host-reachable services
+feature, above. More details can be found in :ref:`kubeproxy-free`, along with
```

Comment:
We can add a link to the specific guide - https://docs.cilium.io/en/v1.9/gettingstarted/host-services/

Reply:
I have a link to that page on L732. Is the preference to re-link it here too, in place of `kubeproxy-free`?

Reply:
If that's the case, can we fold this comment into L729-732?

Reply:
The https://docs.cilium.io/en/v1.9/gettingstarted/host-services/ page should be superseded by the `kubeproxy-free` page, so I would refrain from linking to it.

Reply:
Moved this into the first paragraph. PTAL.